packages/run-protocol/src/vpool-xyk-amm/pool.js

```diff
@@ -114,8 +121,20 @@ const helperBehavior = {
     }
     zcfSeat.exit();
     updateUpdaterState(updater, pool);
+    facets.helper.updateMetrics();
     return 'Added liquidity.';
   },
+  /** @param {MethodContext} context */
+  updateMetrics: context => {
+    const { state, facets } = context;
+    const payload = harden({
+      centralAmount: facets.pool.getCentralAmount(),
+      secondaryAmount: facets.pool.getSecondaryAmount(),
+      liquidityTokens: state.liqTokenSupply,
+    });
+
+    state.metricsPublication.updateState(payload);
```

Comment:
I don't see any change in these values in response to trades.

I added 50 RUN, 50 AUSD of initial liquidity, and then did 2 trades:
 - 25 RUN for 24.27 AUSD
 - 30 RUN for 29.12 AUSD

after each trade, the notifier reported the same values: 50 RUN, 50 AUSD, and 50 liquidity.


![image](https://user-images.githubusercontent.com/150986/169932912-b0ab6f4b-a2b2-428c-a534-8309fcac5a93.png)

Reply:
Those 2 trades didn't actually go through; there was too much slippage. (The misleading feedback is a known issue: #4809).

I traded 0.2 for >= 0.19; this time, I confirmed that my wallet balances changed. And this time, I see updates as I would expect:

```
    Central: { brand: Object [Alleged: RUN brand] {}, value: 50199880n },
    Secondary: { brand: Object [Alleged: AUSD brand] {}, value: 49801395n },
```